### PR TITLE
Replacing binomrand

### PR DESCRIPTION
**Pull request by Ward9250**
Replacing binomrand with using rand method on the binomial type from Distributions.jl

I was unsure on whether using StatsBase needs to be removed or not.

### CODE REVIEW

**Comment by CarloLucibello** [comment]
Can  you also  remove the StatsBase dependence?

**Comment by CarloLucibello** [comment]
... and use a selective import forma binomrand

**Comment by Ward9250** [comment]
StatsBase dependency has been removed now.

@CarloLucibello Forgive me, I'm not sure what you mean by "... and use a selective import forma binomrand" as binomrand is the method that is being replaced.

File: src/LightGraphs.jl [hunk 1]
```diff
@@ -3,7 +3,7 @@ module LightGraphs
 
 using Requires
 using GZip
-using StatsBase: binomrand
+using Distributions
```
Comment (pranavtbhat) [inline src/LightGraphs.jl]: I think @CarloLucibello wants this to be `using Distributions: Binomial`

Comment (sbromberger) [inline src/LightGraphs.jl]: Yes, that's good practice and we should be encouraging explicit imports going forward.

Comment (Ward9250) [inline src/LightGraphs.jl]: Ah I see, ok I'll just do that now.

**Comment by sbromberger** [comment]
Thanks!

**Comment by Ward9250** [comment]
No problem. Do you know when the next tag is expected for LightGraphs? I guess this will still cause errors for some until the next tag, unless they Pkg.clone instead of Pkg.add.

**Comment by sbromberger** [comment]
Soon. I'd like to get triangles in there (#378) and then I'll go ahead and tag.

**Comment by sbromberger** [comment]
New tags!

**Pull request merged**